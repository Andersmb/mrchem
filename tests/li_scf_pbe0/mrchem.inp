# vim:syntax=sh:

world_prec = 1.0e-3               # Overall relative precision
world_size = 5                    # Size of simulation box 2^n

MPI {
  numerically_exact = true        # Guarantee identical results in MPI
}

Molecule {
multiplicity = 2
$coords
Li   0.0     0.0      0.0
$end
}

WaveFunction {
  method = PBE0                   # Wave function method (HF or DFT)
  restricted = false
}

Properties {
  scf_energy = true               # Compute ground state energy
}

SCF {
  kain = 3                        # Length of KAIN iterative history
  max_iter = 5
  orbital_thrs = 2.0e-2
  initial_guess = GTO             # Type of initial guess (none, gto, mw)
}

