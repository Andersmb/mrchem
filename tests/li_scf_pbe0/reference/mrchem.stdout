


************************************************************
***     __  __ ____   ____ _                             ***
***    |  \/  |  _ \ / ___| |__   ___ _ __ ___           ***
***    | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \          ***
***    | |  | |  _ <| |___| | | |  __/ | | | | |         ***
***    |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|         ***
***                                                      ***
***    VERSION 0.0.1 (rev. 996f83d5)                      ***
***                                                      ***
***    Stig Rune Jensen <stig.r.jensen@uit.no>           ***
***    Luca Frediani    <luca.frediani@uit.no>           ***
***    Peter Wind       <peter.wind@uit.no>              ***
***                                                      ***
************************************************************


+++ Serial execution


------------------------------------------------------------

 MRCPP version   : X.Y.Z
 Git revision    : e5c44e2

 Linear algebra  : EIGEN
 Parallelization : NONE

------------------------------------------------------------




============================================================
                  MultiResolution Analysis                  
------------------------------------------------------------
 polynomial order = 5
 polynomial type  = Interpolating
------------------------------------------------------------
 unit length      = 16.000000
 total boxes      = 8
 boxes            = [           2           2           2 ]
 lower bounds     = [  -16.000000  -16.000000  -16.000000 ]
 upper bounds     = [   16.000000   16.000000   16.000000 ]
 total length     = [   32.000000   32.000000   32.000000 ]
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     Li          0.00000e+00   0.00000e+00   0.00000e+00
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================


============================================================
                Setting up nuclear potential
------------------------------------------------------------
 Nr  Element         Charge        Precision     Smoothing 
------------------------------------------------------------
  1     Li           3.00000e+00   1.00000e-03   2.61594e-03
------------------------------------------------------------
 Projecting potential                  840      1.84329e-01
============================================================


============================================================
                Setting up Gaussian-type MOs
------------------------------------------------------------
    n  Spin  Occ                           SquareNorm
------------------------------------------------------------
    0    a    1                       9.999999988178118e-01
    1    a    1                       1.000000048121802e+00
------------------------------------------------------------
                 Wall time: 1.90745e-01 sec 
============================================================


============================================================
                Setting up Gaussian-type MOs
------------------------------------------------------------
    n  Spin  Occ                           SquareNorm
------------------------------------------------------------
    0    b    1                       9.999999988146170e-01
------------------------------------------------------------
                 Wall time: 1.05025e-01 sec 
============================================================


============================================================
 OrbitalVector:   3 orbitals     3 occupied     3 electrons
------------------------------------------------------------
   n  RankID           Norm          Spin Occ      Error    
------------------------------------------------------------
   0     0       9.999999988178e-01    a   1    1.00000e+00
   1     0       1.000000048122e+00    a   1    1.00000e+00
   2     0       9.999999988146e-01    b   1    1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Coulomb density                       392      2.55596e-01
 Coulomb potential                     200      1.77039e+00
 XC alpha density                      392      1.73946e-01
 XC beta density                       392      8.89597e-02
 XC evaluate xcfun                    3528      3.14018e+00
 XC energy                             392      3.26130e-05
 XC potential                          784      1.07124e-01
 XC grid size                          392           594432
 XC grid change                       -384          -580608
 Hartree-Fock exchange                1112      7.18260e+00
------------------------------------------------------------
                 Wall time: 1.28004e+01 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         3.678816720000000e-01
 Potential part                       1.139049400000000e+00
------------------------------------------------------------
                 Wall time: 1.50711e+00 sec 
============================================================




####################### SCF cycle  1 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              2.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        1.05075e-03
 Diagonalizing matrix                           1.27440e-05
 Rotating orbitals                              3.39029e-02
------------------------------------------------------------
                 Wall time: 3.50417e-02 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       0.000000000000000e+00
 Electronic energy                   -7.413819949580088e+00
------------------------------------------------------------
 Total energy                        -7.413819949580088e+00
------------------------------------------------------------
                 Wall time: 1.13889e+00 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.00369e+00   328    0.00000e+00     0   3.68024e+00
  1    9.97845e-01   264    0.00000e+00     0   3.55549e+00
  2    1.00480e+00   328    0.00000e+00     0   3.47004e+00
------------------------------------------------------------
                 Wall time: 1.07061e+01 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.28260e-05
 Copy orbitals                                  1.20445e-02
 Copy orbital updates                           1.55409e-02
------------------------------------------------------------
                 Wall time: 2.76949e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Coulomb density                       392      2.51035e-01
 Coulomb potential                     200      1.76996e+00
 XC alpha density                      392      1.72298e-01
 XC beta density                       392      9.02008e-02
 XC evaluate xcfun                    3528      3.13073e+00
 XC energy                             392      1.82620e-05
 XC potential                          784      1.06307e-01
 XC grid size                          392           594432
 XC grid change                       -384          -580608
 Hartree-Fock exchange                1176      7.10043e+00
------------------------------------------------------------
                 Wall time: 1.27026e+01 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         3.772165070000000e-01
 Potential part                       1.070122770000000e+00
------------------------------------------------------------
                 Wall time: 1.44754e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -2.06049e+00   3.76631e-02        328    a    1    0
  1  -1.41630e-01   4.43976e-02        264    a    1    0
  2  -2.05059e+00   3.85938e-02        328    b    1    0
------------------------------------------------------------
 Total error:                            6.98509e-02    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -4.182018094178723e+00    -4.18202e+00    1
 Kinetic       7.370996345565672e+00     7.37100e+00    1
 N-E          -1.703502983029751e+01    -1.70350e+01    1
 Coulomb       4.061901604356892e+00     4.06190e+00    1
 Exchange     -4.448032885822928e-01    -4.44803e-01    1
 X-C          -1.366884780622850e+00    -1.36688e+00    1
------------------------------------------------------------
 Electronic   -7.413819949580088e+00    -7.41382e+00    1
 Nuclear       0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Total        -7.413819949580088e+00    -7.41382e+00    1
============================================================


################ Wall time: 2.62053e+01 sec ################




####################### SCF cycle  2 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              2.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        1.03890e-03
 Diagonalizing matrix                           6.42500e-06
 Rotating orbitals                              3.24927e-02
------------------------------------------------------------
                 Wall time: 3.36100e-02 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       0.000000000000000e+00
 Electronic energy                   -7.465195706886627e+00
------------------------------------------------------------
 Total energy                        -7.465195706886627e+00
------------------------------------------------------------
                 Wall time: 1.07439e+00 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    9.97094e-01   328    0.00000e+00     0   3.43854e+00
  1    9.96666e-01   264    0.00000e+00     0   3.54940e+00
  2    9.97294e-01   328    0.00000e+00     0   3.41691e+00
------------------------------------------------------------
                 Wall time: 1.04052e+01 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             3.82600e-06
 Copy orbitals                                  1.00776e-02
 Copy orbital updates                           1.16364e-02
------------------------------------------------------------
                 Wall time: 2.18125e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Coulomb density                       392      2.22561e-01
 Coulomb potential                     200      1.75245e+00
 XC alpha density                      392      1.51131e-01
 XC beta density                       392      8.57964e-02
 XC evaluate xcfun                    3528      3.13287e+00
 XC energy                             392      1.83700e-05
 XC potential                          784      1.06621e-01
 XC grid size                          392           594432
 XC grid change                       -384          -580608
 Hartree-Fock exchange                1176      6.74096e+00
------------------------------------------------------------
                 Wall time: 1.22612e+01 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         3.655822000000000e-01
 Potential part                       1.096856784000000e+00
------------------------------------------------------------
                 Wall time: 1.46264e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -2.05590e+00   3.74540e-03        328    a    1    1
  1  -1.41007e-01   1.67366e-02        264    a    1    1
  2  -2.04567e+00   3.66210e-03        328    b    1    1
------------------------------------------------------------
 Total error:                            1.75372e-02    1
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -4.252714179175314e+00    -7.06961e-02    1
 Kinetic       7.311713966078246e+00    -5.92824e-02    1
 N-E          -1.701188463939257e+01     2.31452e-02    1
 Coulomb       4.035279564821570e+00    -2.66220e-02    1
 Exchange     -4.414137204317500e-01     3.38957e-03    1
 X-C          -1.358890877962122e+00     7.99390e-03    1
------------------------------------------------------------
 Electronic   -7.465195706886627e+00    -5.13758e-02    1
 Nuclear       0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Total        -7.465195706886627e+00    -5.13758e-02    1
============================================================


################ Wall time: 2.53958e+01 sec ################


============================================================
                      Convergence rate
------------------------------------------------------------
Iter    OrbError       Property                   Update  
------------------------------------------------------------
  1    6.98509e-02    -7.413819949580088e+00   -7.41382e+00
  2    1.75372e-02    -7.465195706886627e+00   -5.13758e-02
------------------------------------------------------------
                      SCF converged!!!                      
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        1.04815e-03
 Diagonalizing matrix                           6.70300e-06
 Rotating orbitals                              3.66848e-02
------------------------------------------------------------
                 Wall time: 3.78178e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Coulomb density                       392      2.25475e-01
 Coulomb potential                     200      1.74769e+00
 XC alpha density                      392      1.57868e-01
 XC beta density                       392      8.90530e-02
 XC evaluate xcfun                    3528      3.12205e+00
 XC energy                             392      1.93460e-05
 XC potential                          784      1.05610e-01
 XC grid size                          392           594432
 XC grid change                       -384          -580608
 Hartree-Fock exchange                1176      6.70637e+00
------------------------------------------------------------
                 Wall time: 1.22316e+01 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 1.15516e+00 sec 
============================================================


============================================================
                        Fock matrix
------------------------------------------------------------
-2.05590e+00  0.00000e+00  0.00000e+00
 0.00000e+00 -1.41007e-01  0.00000e+00
 0.00000e+00  0.00000e+00 -2.04567e+00
============================================================


============================================================
                      Orbital energies
------------------------------------------------------------
    n  spin  occ                            epsilon  
------------------------------------------------------------
    0    a    1                      -2.055896857655857e+00
    1    a    1                      -2.045667932170998e+00
    2    b    1                      -1.410070339564724e-01
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     Li          0.00000e+00   0.00000e+00   0.00000e+00
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================

                                                            
                                                           
============================================================
                         SCF Energy                         
------------------------------------------------------------
                                                            
 Sum orbital energy:                 -4.242546643343410e+00
 Kinetic energy:                      7.380315908889136e+00
 E-N energy:                         -1.708858675414508e+01
 Coulomb energy:                      4.048568577039632e+00
 Exchange energy:                    -4.427583475848431e-01
 X-C energy:                         -1.363557098320302e+00
 El. external field energy:           0.000000000000000e+00
 Nuc. external field energy:          0.000000000000000e+00
                                                            
------------------------------------------------------------
                                                            
 Electronic energy:                  -7.466017714121453e+00
 Nuclear energy:                      0.000000000000000e+00
                                                            
------------------------------------------------------------
                                                            
 Total energy       (au)             -7.466017714121453e+00
                    (kJ/mol)         -1.960202667133915e+04
                    (kcal/mol)       -4.684996811332947e+03
                    (eV)             -2.031606828090797e+02
                                                            
============================================================
                                                            



************************************************************
***                                                      ***
***                    Exiting MRChem                    ***
***                                                      ***
***               Wall time: 7.992146e+01                ***
***                                                      ***
************************************************************


